From 3770f6c8a5dd37c5fbf1467edb047042ea2791ed Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Wed, 25 Oct 2017 15:40:16 +0800
Subject: [PATCH] arm64: rockchip_defconfig: disable DEVKMEM and DEVPORT

42019da52a83 ("android: base-cfg: turn off /dev/mem and /dev/kmem")
870f86b79c85 ("Unsetting DEVPORT from marlin configurations.")

Change-Id: I03610d55f3af2e9398b02e36d1d6c05a2eda9657
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/configs/rockchip_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/rockchip_defconfig b/arch/arm64/configs/rockchip_defconfig
index 1076ac39c14b..cc1e736f2714 100644
--- a/arch/arm64/configs/rockchip_defconfig
+++ b/arch/arm64/configs/rockchip_defconfig
@@ -387,6 +387,7 @@ CONFIG_INPUT_GPIO=y
 # CONFIG_SERIO is not set
 # CONFIG_VT is not set
 # CONFIG_LEGACY_PTYS is not set
+# CONFIG_DEVKMEM is not set
 CONFIG_SERIAL_8250=y
 # CONFIG_SERIAL_8250_DEPRECATED_OPTIONS is not set
 CONFIG_SERIAL_8250_CONSOLE=y
@@ -395,6 +396,7 @@ CONFIG_SERIAL_8250_NR_UARTS=5
 CONFIG_SERIAL_8250_RUNTIME_UARTS=5
 CONFIG_SERIAL_8250_DW=y
 # CONFIG_HW_RANDOM is not set
+# CONFIG_DEVPORT is not set
 CONFIG_I2C_CHARDEV=y
 CONFIG_I2C_GPIO=y
 CONFIG_I2C_RK3X=y
-- 
2.35.3

